packages-next/fields-document/ARCHITECTURE.md

```diff
@@ -0,0 +1,53 @@
+# Document editor architecture
+
+The document editor is using [Slate](https://github.com/ianstormtaylor/slate). To learn about Slate generally, go through https://docs.slatejs.org/walkthroughs and https://docs.slatejs.org/concepts. The sections on [Nodes](https://docs.slatejs.org/concepts/02-nodes) and [Locations](https://docs.slatejs.org/concepts/03-locations) contain especially important terminology.
+
+## Plugins
+
+For normalization and handling user input and some other things(you can see what they are by looking at the type definition for the `Editor` in Slate), you write [Slate "plugins"](https://docs.slatejs.org/concepts/07-plugins), plugins are not some special API, it means accepting an `Editor` object and returning it but modifying some of the properties.
+
+## Rendering
+
+The rendering of the editor is mostly like any other bit of React code. See https://docs.slatejs.org/concepts/08-rendering for how rendering the document works in Slate.
```

Comment:
> mostly like any other bit of React code

Are there specific differences we should know about?

Reply:
Besides the performance stuff and the stuff explained in the Slate docs, no

Reply:
In that case can we remove the word `mostly`? It makes me wonder if there are specific things I need to be looking for.